docs/collector-limitations.md

```diff
@@ -0,0 +1,8 @@
+# Elastic Distribution for OpenTelemetry Collector limitations
+
+The Elastic OTel Collector has the following limitations:
+
+- Because of an upstream limitation, `host.network.*` metrics aren't present from the OpenTelemetry side.
+- `process.state` isn't present in the OpenTelemetry host metric. It's set to a dummy value of **Unknown** in the **State** column of the host processes table.
+- The Elasticsearch exporter handles the metadata fields, but **Host OS version** and **Operating system** may show as "N/A" and **Host IP** may show different values.

```

Comment:
```suggestion
- The Elasticsearch exporter handles the resource attributes, but **Host OS version** and **Operating system** may show as "N/A" and **Host IP** may show different values.
```

`show different values` ... different values to what?

Reply:
We can remove part about host.ip's in this documentation. 
That was our way of recording how Elastic reports Host.ip and how OTEL reports host.ip.
Not relevant to this limitation document